mbuild/lib/recipes/polymer.py

```diff
@@ -83,7 +88,12 @@ def __init__(self, monomers=None, end_groups=None):
         super(Polymer, self).__init__()
         self._monomers = monomers or []
```

Comment:
Add placeholders for `self._head_port` and `self._tail_port` with `None` assigned initially